Misc/NEWS.d/next/Library/2022-08-06-12-18-07.gh-issue-88863.NnqsuJ.rst

```diff
@@ -0,0 +1,3 @@
+To resolve the memory leak when asyncio.open_connection raises, break
+reference cycles generated by local exception and future instances
+(which has exception instance as its member var). Patched by Dong Uk, Kang.
```

Comment:
```suggestion
To avoid apparent memory leaks when :func:`asyncio.open_connection` raises,
break reference cycles generated by local exception and future instances
(which has exception instance as its member var). Patch by Dong Uk, Kang.
```